src/common/json.cc

```diff
@@ -585,39 +686,23 @@ class GlobalCLocale {
   }
 };
 
-Json Json::Load(StringView str, bool ignore_specialization) {
+Json Json::Load(StringView str) {
   GlobalCLocale guard;
-  LOG(WARNING) << "Json serialization is still experimental."
```

Comment:
Why are we removing this warning?

Reply:
@hcho3 Will put it into `Learner::Load` and `Save`.  This implementation will also be used to consume `__array_interface__` from numpy and `__cuda_array_interface__` from cudf and numba and friends.